### PR TITLE
Template update

### PR DESCRIPTION
**Pull request by BATMAN-JD**
## Changes introduced with this PR

Update to latest template standards.

---
By contributing to this repository, I agree to the [contribution guidelines](https://github.com/arcalot/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,56 +1,53 @@
 ARG package=arcaflow_plugin_uperf
 
-# build poetry
-FROM quay.io/centos/centos:stream8 as poetry
+# STAGE 1 -- Build module dependencies and run tests
+# The 'poetry' and 'coverage' modules are installed and verson-controlled in the
+# quay.io/arcalot/arcaflow-plugin-baseimage-python-buildbase image to limit drift
+FROM quay.io/arcalot/arcaflow-plugin-baseimage-python-buildbase:0.2.0 as build
 ARG package
 RUN dnf module -y install python39 && dnf install --setopt=tsflags=nodocs -y python39 python39-pip lksctp-tools-devel && dnf clean all \
```
Comment (dustinblack) [inline Dockerfile]: Installation of python module and packages should be removed

File: Dockerfile [hunk 2]
```diff
@@ -1,56 +1,53 @@
...
+RUN python -m coverage run tests/test_${package}.py \
+ && python -m coverage html -d /htmlcov --omit=/usr/local/*
 
 
-# final image
-FROM quay.io/centos/centos:stream8
+# STAGE 2 -- Build final plugin image
+FROM quay.io/arcalot/arcaflow-plugin-baseimage-python-osbase:0.2.0
 ARG package
 RUN dnf module -y install python39 && dnf install --setopt=tsflags=nodocs -y python39 python39-pip lksctp-tools-devel && dnf clean all \
```
Comment (dustinblack) [inline Dockerfile]: Installation of python module and packages should be removed

**Review by dustinblack — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (BATMAN-JD) [reply Dockerfile]: Removed

File: Dockerfile (hunk 2 above)
Comment (BATMAN-JD) [reply Dockerfile]: Removed

**Review by dustinblack — APPROVED** [review]

**Pull request merged**